app/editor/src/features/admin/filters/constants/defaultFilter.ts

```diff
@@ -20,6 +20,7 @@ export const defaultFilter: IFilterModel = {
     contentTypes: [],
     tags: [],
     sentiment: [],
+    hasTopic: false,
```

Comment:
This should default to `undefined`.  That way a search can be `true` or `false` condition, and an `undefined` will not add this as part of the search.

Reply:
Updated, thanks